docs/resources/extract-ai.sh

```diff
@@ -6,26 +6,88 @@ BINARY_FOLDER=/var/mnt
 
 pushd $FOLDER
 
-total_copies=$(sort -u $BINARY_FOLDER/$OCP_RELEASE_LIST | wc -l)  # Required to keep track of the pull task vs total
-current_copy=1
+load_images() {
```

Comment:
We should consider adding bashate and shellcheck to the tests, similar to what we have in openshift-kni/cluster-group-upgrades-operator. There's nothing serious being raised for this PR, though, so we can do that as a future work item.